src/content/en/tools/workbox/guides/service-worker-checklist.md

```diff
@@ -0,0 +1,65 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/next/_book.yaml
+description:A checklist of things to ensure when setting up a service worker.
+
+{# wf_updated_on: 2017-11-15 #}
+{# wf_published_on: 2017-11-15 #}
+
+# Service Worker Checklist {: .page-title }
+
+This page contains a couple of tips that will ensure your users get the
+best experience from your web app as well as ensuring there aren't any
+hidden suprises while developing your site.
+
+## Register your service worker the right way
+
+In general, you should wait for the window load event before registering your
+service worker. This will allow the browser to prioritise assets for the page
+and will prevent any risk of precaching interferring with the page.
+
+
+```js
+if ('serviceWorker' in navigator) {
+  window.addEventListener('load', function() {
+    navigator.serviceWorker.register('/service-worker.js');
+  });
+}
+```
+
+<div class="success">
+  <b>DO</b>: Wait until the `window` load event to register your service worker.
+</div>
+
+## Cache-Control of your service worker file
+
+When you register your service worker file, the browser will make a network
+request for the file. If your server returns a `Cache-Control` header with
+your service worker file, the browser will
```

Comment:
Only Chrome that will do this, as Firefox 57+ [fixed](https://stackoverflow.com/questions/38843970/service-worker-javascript-update-frequency-every-24-hours/38854905#38854905) this behavior to adhere to the current specification.

I'm not sure how to best capture that nuance... I guess rephrase to "...some browsers will..."